tests/conftest.py

```diff
@@ -69,3 +73,12 @@ def randomsizedarray():
     a = random.randint(2, 8)
     b = random.randint(2, 8)
     return np.random.random((a, b))
+
+
+@pytest.fixture()
+def get_history(scope='sesssion'):
```

Comment:
Please don't define this as a fixture function, it's a built in utility of the library, and should just be called directly from a repository object in any tests which need it via the method `repo.summary()` (optionally with `return_contents=True` for programatic evaluation.  No need to make test fixtures more complex then they already area :)

Reply:
I was thinking we'll keep all the dependencies to fixtures and keep the `import`s minimal as possible (Just another way of test designs we used to follow). Let's do it your way